docs/proposals/karmadactl/Karmadactl Supports Configuration File Method for Production-Grade Installation and Deployment.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Karmadactl Supports Configuration File Method for Production-Grade Installation and Deployment
+authors:
+  - "@tiansuo114"
+reviewers:
+  - "@liangyuanpeng"
+
+approvers:
+  - "@liangyuanpeng"
+
+creation-date: 2024-07-29
+---
+
+# Karmadactl Supports Configuration File Method for Production-Grade Installation and Deployment
+
+## Summary
+
+Additionally, `karmadactl init` requires the use of multiple container images during deployment, and users currently cannot easily obtain a list of the container images used. Therefore, we plan to add an `images list` subcommand to allow users to print the list of container images used during the `karmadactl init` process to the console.
```

Comment:
wo can add some summary about supporting configuration files.

Reply:
Ok, I have modified the section and the following questions
@zhzhuang-zju